src/components/modules/banners/environmentMomentBanner/components/EnvironmentMomentBannerBody.tsx

```diff
@@ -2,45 +2,68 @@ import React from 'react';
 import { css } from '@emotion/core';
 import { body } from '@guardian/src-foundations/typography';
 import styles from '../helpers/styles';
+import { getLocalCurrencySymbol } from '../../../../../lib/geolocation';
 
 const container = css`
     ${body.small()}
 `;
 
+const paragraphsContainer = css`
+    & > * + * {
+        margin-top: 8px;
+    }
+`;
+
 interface EnvironmentMomentBannerBodyProps {
     isSupporter: boolean;
+    countryCode: string;
```

Comment:
not important, but this field can be optional because `getLocalCurrencySymbol` defaults it to `£`

Reply:
I'll make this change anyways! It only takes 2 secs and I like it 👍